android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsCommon.java

```diff
@@ -178,8 +178,16 @@ public static AdManagerAdRequest buildAdRequest(ReadableMap adRequestOptions) {
 
       for (Map.Entry<String, Object> entry : customTargeting.entrySet()) {
         String key = entry.getKey();
-        String value = (String) entry.getValue();
-        builder.addCustomTargeting(key, value);
+        Object value = entry.getValue();
+        String type = value.getClass().getSimpleName();
+
+        if (type.equals("String")) {
```

Comment:
Instead of getting the type I believe you can just check if it's an instanceof String.
`if (value instanceof String) {`